### ISSUE TITLE
CAF 2011-15

### ISSUE THREAD

**Issue by eclee25** [HopkinsIDD/cholera-mapping-pipeline#383]

**Comment by QLLZ** [comment]
Data pull:
HASH:  `6772ea9da4f43ddc3da11c494a573af2ca3dd3c8`
[config](https://github.com/HopkinsIDD/cholera-configs/tree/master/no_covariate_production_2023/2011_2015_country)

**Comment by QLLZ** [comment]
Model run:
HASH: `6772ea9da4f43ddc3da11c494a573af2ca3dd3c8`

**Comment by QLLZ** [comment]
[Country data report](https://github.com/HopkinsIDD/cholera-mapping-reports/tree/2023_nocovar_production_runs/country_data_reports)

**Comment by eclee25** [comment]
Rhats look good, convergence is pretty good, with slight instability in sd_w and w. There are no non-zero obs in 2013-15 and observation fits look good. The gridded outputs seems to be suffering from some inflated cases and rates on the eastern border, which I suspect may be related to sfrac. 

Opinion: Investigate border cell sfracs and estimates, but otherwise I think this is ok.

**Comment by javierps** [comment]
Agreed that the edge effects are not ideal. At the same time the mean modeled number of cases is around 1-10, so I don't think it is worrying.

Would suggest accepting this.

**Comment by javierps** [comment]
May consider a no-w run given the small amount of subnational level data.

**Comment by QLLZ** [comment]
[Updated config](https://github.com/HopkinsIDD/cholera-configs/blob/master/no_covariate_production_2023/round3/no_w/config_prd-2023-rd3_CAF_2011_2015.yml)

**Comment by QLLZ** [comment]
Data pull + model run:
HASH: `20c484bb812a2bc77312f65157b638d040e72487`

**Comment by QLLZ** [comment]
[Country data report](https://github.com/HopkinsIDD/cholera-mapping-reports/blob/2023_nocovar_production_runs/country_data_reports/CAF_2011-01-01_2015-12-31_prd-2023-rd3_68427d7bd7822a059e0329ce5d88c6da.country-data-report.html)

**Comment by eclee25** [comment]
ptc ID 45 with dev_dagar_fix and a spatial effect looks good. There is some subnational data in this model run.
Opinion: Approve

**Comment by QLLZ** [comment]
Data pull and model run (dev_taxdat_funs_fix branch):

HASH: `31f14227cd28a9824db72b7a6c45b0d65652872e`

**Comment by QLLZ** [comment]
log files were accidently deleted.

**Comment by QLLZ** [comment]
[country data report](https://github.com/HopkinsIDD/cholera-mapping-reports/blob/July_2023_nocovar_production_runs/country_data_reports/CAF_2011-01-01_2015-12-31_prd-jul2023_a791ce9bcab1059b5f612b67091627ab.country-data-report.html)

**Comment by QLLZ** [comment]
All look good.

Suggestion: approve.

**Comment by eclee25** [comment]
Minimal cases and subnational data. All diagnostics good (standard model)
Approve.

**Comment by eclee25** [comment]
Slight instability in sd_w and w traceplots with a high sd_w value. Probably could be improved with no-mixture sd_w prior.

**Comment by QLLZ** [comment]
Data pull, model run:

HASH: `05f598b95b5d42ac0049963f8a0b7e04d6f1dbd7`

**Comment by javierps** [comment]
All OK.

Suggestion: Accept.

**Comment by eclee25** [comment]
Agree with J. Approve
